src/site/stages/build/drupal/health-care-region.js

```diff
@@ -297,6 +269,39 @@ function addGetUpdatesFields(page, pages) {
   }
 }
 
+/**
+ * Sorts items from oldest to newest, removing expired items.
+ *
+ * @param {items} object The items object.
+ * @param {field} string The target date field.
+ * @param {reverse} bool Sorting order set to default false.
+ * @param {stale} bool Remove expired date items set to default false.
+ * @return Filtered array of sorted items.
+ */
+function itemSorter(items, field, reverse = false, stale = true) {
+  const dateFiltered = [];
+  const sorted =
+    items !== null
+      ? items.entities.sort((a, b) => {
+          const aTime = Math.floor(new Date(a[field].value).getTime() / 1000);
+          const bTime = Math.floor(new Date(b[field].value).getTime() / 1000);
+          // Sort order.
+          const sorter = reverse ? bTime - aTime : aTime - bTime;
+          return sorter;
+        })
+      : '';
+  // Remove expired items.
+  sorted.forEach(element => {
+    if (
+      element[field] &&
+      new Date(element[field].value).valueOf() > new Date().valueOf()
+    ) {
+      dateFiltered.push(element);
+    }
+  });
+  return stale ? dateFiltered : sorted;
+}
+
```

Comment:
This doesn't read super well for me either. I see the `sorted = items !== null` safety check, but then we do `sorted.forEach`, so it would just hit a runtime error there anyway.

We have `moment.js` at our disposal, so might as well make use of it -

```js
const moment = require('moment');

function itemSorter(items = [], field, reverse = false, stale = true) {
  let sorted = items.sort((a, b) => {
     const start1 = moment(a[field].value);
     const start2 = moment(b[field].value);
     return start1.isAfter(start2);
  });

  if (reverse) {
    sorted = sorted.reverse();
  }

  if (stale) {
    sorted = sorted.filter(item => moment(item[field]).isAfter(moment());
  }

  return sorted;
}
```

Reply:
:+1: @ncksllvn